packages/syntax-core/src/LabeledCheckbox/LabeledCheckbox.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactElement, useState } from "react";
+import classNames from "classnames";
+import styles from "./LabeledCheckbox.module.css";
+import Typography from "../Typography/Typography";
```

Comment:
Should we use absolute imports everywhere instead?

Reply:
@JFChoe Not in Syntax - we still have to set up absolute imports